apps/webservice/src/app/[workspaceSlug]/(targets)/target-metadata-groups/EditMetadataGroupDialog.tsx

```diff
@@ -177,13 +130,18 @@ export const UpsertMetadataGroupDialog: React.FC<{
 
             <div>
               <Label>Keys</Label>
+              {fields.length === 0 && (
+                <p className="h-[30px] text-xs text-muted-foreground">
```

Comment:
added this exact amount so that the height of the dialog wouldn't change slightly when you add first key